src/kagglehub/exceptions.py

```diff
@@ -53,3 +53,11 @@ def kaggle_api_raise_for_status(response: requests.Response):
 
         # Default handling
         raise KaggleApiHTTPError(message, response=response) from e
+
+
+def postprocess_response(response: dict) -> None:
```

Comment:
nit: We only use this pattern of returning the error in our `POST` RPCs.

Let's call this method `process_post_response` instead.